pandas/tests/io/formats/test_to_csv.py

```diff
@@ -561,3 +563,21 @@ def test_to_csv_compression(self, compression_only,
             result = pd.read_csv(path, index_col=0,
                                  compression=read_compression)
             tm.assert_frame_equal(result, df)
+
+    @pytest.mark.parametrize("df_value", [float('nan')])
```

Comment:
If we are only passing single values here do we even need parametrization?

Reply:
I think the idea was to add more values later, but maybe I never finished the whole work so we didn't get to the point of discussing which new parameters to use?

See https://github.com/pandas-dev/pandas/pull/25103#discussion_r254126828

Reply:
From that comment I think you just need one parametrization here for dtype, so something like this:

```python
@pytest.mark.parametrize("dtype", [float, 'Int64'])
```

And then do your data frame construction as follows:

```python
df = pd.DataFrame([np.nan] * 3, columns=['c'], dtype=dtype)
```

No need for the astype call thereafter

Reply:
Makes sense! Change applied, code rebased. Thanks!